plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -231,9 +231,9 @@ export const CITableView = ({
       onChangeRowsPerPage={onChangePageSize}
       title={
         <Box display="flex" alignItems="center">
-          <GitHubIcon />
-          <Box mr={1} />
-          <Typography variant="h6">{projectName}</Typography>
+          <img src={JenkinsLogo} alt="Jenkins logo" height="50px" />
```

Comment:
I know this is a bit of a hack, so open to other approaches to do this. I've seen people write a const anonymous funcs and wrap it all so you can one time call something like `<JenkinsLogo />` but thought that might be overkill here. And didn't bother extracting the height out to a style, either. It works, even if brute force ☺️

Reply:
This is fine!

Reply:
as for alternatives ... i dunno, maybe a ListItem with a ListItemAvatar + Avatar? :) https://material-ui.com/components/lists/ If you like to experiment